ecs.tf

```diff
@@ -4,7 +4,7 @@ data "aws_ecs_cluster" "target_cluster" {
 
 resource "aws_ecs_task_definition" "main_task" {
   family                   = "${var.app_name}-tsk"
-  requires_compatibilities = ["EC2"]
+  requires_compatibilities = [var.launch_type]
```

Comment:
This variable needs to be declared in the varaibles.tf file.

Reply:
It's actually already there: 
`variable "launch_type" {
  default     = "EC2"
  description = "The launch type for the task. We assume EC2 by default."
}`